### ISSUE TITLE
DuckDB in DBeaver (JDBC) doesn't show information_schema tables/views

### ISSUE THREAD

**Issue by Alex-Monahan** [duckdb/duckdb#2809]
Hi Folks!

As pathfinding for a future blog post, I got DBeaver set up with DuckDB. It is super slick - it took like 2 minutes. I think it will make a create "speed run" little tutorial video.

One small gap is that while DBeaver shows the information_schema, it does not have the tables, views, or functions populated. I think those could come in handy for folks when they want to explore their DB's metadata and could make for a nice initial/demo experience as well.

This shows that the tables and views sections are empty (with no drilldown arrow next to them) for the metadata schemas.
![image](https://user-images.githubusercontent.com/52226177/146564731-0f189f0d-8120-48c9-a8f0-f6d87008649f.png)

Thanks!
-Alex

**Comment by Mytherin** [comment]
Are views shown in general? The information_schema/pg_catalog schemas only contain views, so perhaps the problem is with that.

Is this using the JDBC connector, or another connector?

**Comment by Alex-Monahan** [comment]
Yes, views can be shown, I just didn't create one in my example - sorry!
It's using the JDBC connector

**Comment by Mause** [comment]
@Alex-Monahan can you check if this is still the case?

**Comment by Alex-Monahan** [comment]
> @Alex-Monahan can you check if this is still the case?

I'll have a look!

**Comment by franemar** [comment]
With DBeaver version 23.04 and DuckDB driver v0.71 custom tables are been showing in the UI: 
![image](https://user-images.githubusercontent.com/30542217/236833145-92546879-ed70-47fa-bd56-afe82e5364b7.png)
 `information_schema` is still empty in the UI, but its views are available in the SQL script:
![image](https://user-images.githubusercontent.com/30542217/236833398-94c5e839-86e3-44db-a7a4-e91167e4a558.png)
![image](https://user-images.githubusercontent.com/30542217/236834346-d0d94637-cc7d-47e2-94fe-840865d3d87c.png)
So, this driver version seems to be working fine and there is a pending implementation in the DBeaver's UI.

**Comment by jalvarezcabada** [comment]
Hi everyone, I have the same issue in DataGrip version 2023.2. I can't see the tables in the `main` schema.
![image](https://github.com/duckdb/duckdb/assets/110677560/a25f1d1a-35a9-4063-8895-db89e6bc2940)

**Comment by LonwoLonwo** [comment]
> As shown in https://github.com/duckdb/duckdb/issues/2809#issuecomment-1538353037, DBeaver now displays DuckDB tables correctly.

Hello @szarnyasg 

The initial issue was about information_schema tables/views. Not about the custom tables.

![2024-02-28 14_17_55-DBeaver Ultimate 24 0 0 - _duck_010 db_ Script-178](https://github.com/duckdb/duckdb/assets/45152336/72904cf9-c1ef-44c7-8bf3-67186bf5f2a9)

The issue is still here, and I want to know how we can fix it.

>  I think those could come in handy for folks when they want to explore their DB's metadata 

Because I am one of these people :D

Please reopen this issue.
